### PR TITLE
Bl garnett immersive

### PR DESCRIPTION
**Pull request by blongden73**
## What does this change?
Immersive design tweaks. Without touching the gallery and photo essay. 

@regiskuckaertz @AWare @GHaberis

### CODE REVIEW

File: common/app/views/fragments/photoEssay.scala.html [hunk 1]
```diff
@@ -12,7 +12,13 @@
     page.item.tags.isPaidContent
 )) { case (hasEmbed, hasMainMedia, hasVideo, isPaidContent) =>
   <style>
+  @if(!experiments.ActiveExperiments.isParticipating(experiments.Garnett)) {
     @Html(common.Assets.css.inlinePhotoEssay)
+  }
```
Comment (AWare) [inline common/app/views/fragments/photoEssay.scala.html]: This could be an else.

**Comment by NataliaLKB** [comment]
This branch needs to be rebased or merged with master (thats why the tests are failing)

File: article/app/pages/ArticleHtmlPage.scala [hunk 2]
```diff
@@ -15,7 +15,12 @@ object ArticleHtmlPage extends HtmlPage[ArticlePage] {
 
     val (header, content): (Html, Html) = page.article match {
       case article if article.isPhotoEssay => (photoEssayHeader(), immersiveBody())
+      case article if article.isImmersive && experiments.ActiveExperiments.isParticipating(experiments.Garnett) => (
+          immersiveGarnettHeader(), immersiveGarnettBody()
+      )
       case article if article.isImmersive => (immersiveHeader(), immersiveBody())
+
+
```
Comment (NataliaLKB) [inline article/app/pages/ArticleHtmlPage.scala]: Nit: can we remove these extra unused lines?

File: article/app/views/fragments/immersiveGarnettBody.scala.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+@import views.support.TrailCssClasses.toneClass
+@import experiments._
+@import _root_.model.ContentDesignType.RichContentDesignType
+
+@defining(model.article) { article =>
+  @defining(isPaidContent(model)) { isPaidContent =>
+        <article id="article" data-test-id="article-root"
+
+            class="@RenderClasses(Map(
+                "has-feature-showcase-element" -> (article.tags.isFeature && article.elements.hasShowcaseMainElement),
```
Comment (NataliaLKB) [inline article/app/views/fragments/immersiveGarnettBody.scala.html]: Does immersive do something different in a showcase now?

File: article/app/views/fragments/immersiveGarnettBody.scala.html [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+@import views.html.fragments.langAttributes
+@import common.LinkTo
+@import views.BodyCleaner
+@import views.support.RenderClasses
+@import views.support.Commercial.{isPaidContent, articleAsideOptionalSizes, shouldShowAds}
+@import views.support.TrailCssClasses.toneClass
+@import experiments._
+@import _root_.model.ContentDesignType.RichContentDesignType
+
+@defining(model.article) { article =>
```
Comment (NataliaLKB) [inline article/app/views/fragments/immersiveGarnettBody.scala.html]: You could do this instead

```
@defining((model.article, isPaidContent(model))) { case (article, isPaidContent) =>
```

File: static/src/stylesheets/module/content-garnett/_article-immersive.scss [hunk 5]
```diff
@@ -567,3 +560,191 @@
...
+            min-height: 600px;
+        }
+    }
+    .content__article-body {
+        @include mq($from: desktop) {
+            margin-left: -$gs-gutter / 2;
+        }
+    }
+    .immersive-main-media__headline-container {
+        margin-top: -($gs-column-width * 2) + 4px;
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content-garnett/_article-immersive.scss]: ` -($gs-column-width * 2) + 4px` ? lets just say `-124px`. I don't like the magic numbers, but at least lets be clear about it 😄

File: static/src/stylesheets/module/content-garnett/_article-immersive.scss [hunk 6]
```diff
@@ -567,3 +560,191 @@
...
+        }
+
+        .content__main-column {
+            background-color: $garnett-neutral-1;
+            z-index: 10;
+            position: relative;
+            padding-left: $gs-baseline;
+            left: -$gs-baseline / 2;
+            padding-top: $gs-baseline / 2;
+            width: 94%;
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content-garnett/_article-immersive.scss]: Why 94%? Can we add a comment?

File: static/src/stylesheets/module/content-garnett/_article-immersive.scss [hunk 7]
```diff
@@ -567,3 +560,191 @@
...
+
+        @include mq(desktop) {
+            max-width: 460px;
+            padding-bottom: 1rem;
+            margin-bottom: -$gs-baseline / 2;
+            margin-top: $gs-baseline / 2;
+        }
+
+        &:before {
+            position: absolute;
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content-garnett/_article-immersive.scss]: Can we add a comment to say what this is?

File: article/app/views/fragments/immersiveGarnettBody.scala.html (hunk 3 above)
Comment (paperboyo) [reply article/app/views/fragments/immersiveGarnettBody.scala.html]: 🤔

**Review by NataliaLKB — APPROVED** [review]
Great! Thanks @blongden73

File: article/app/views/fragments/immersiveGarnettBody.scala.html (hunk 3 above)
Comment (blongden73) [reply article/app/views/fragments/immersiveGarnettBody.scala.html]: @paperboyo No...this was just something I missed and should have removed.

**Pull request merged**